src/utility/resetStatus.js

```diff
@@ -2,9 +2,10 @@ import { playerStatus } from '../content/playerStatus';
 import { initialStatus } from '../content/initialStatus';
 
 const resetStatus = () => {
-  Object.keys(playerStatus).forEach((key) => {
-    playerStatus[key] = initialStatus[key];
+  Object.keys(playerStatus).forEach((param) => {
+    Array.isArray(playerStatus[param]) ? playerStatus[param] = []
```

Comment:
Don't use the assignment operator as a part of the ternary operator. (`foo ? bar = 1 : bar = 2`)
You should assign the result of it instead. (`bar = foo ? 1 : 2`)